frontend/src/forms/subforms/AddressForm.tsx

```diff
@@ -88,7 +88,7 @@ const AddressForm = <T extends any>(props: AddressProps & FormikProps<T>) => {
             formikProps={props}
             disabled={props.disabled}
             outerClassName="col-md-10"
-            onBlurFormatter={(postal: string) => postal.replace(/ /g, '')}
+            onBlurFormatter={(postal: string) => postal.replace(/ /g, '').replace(/-/g, '')}
```

Comment:
I think a single regex like: `[\s-]+` here would avoid the performance hit of executing two regex queries on the string.

https://stackoverflow.com/questions/25889199/regex-for-replacing-multiple-spaces-and-dashes-with-or-without-spaces

Reply:
Thank you! I think this will also help another issue I was having